src/template.js

```diff
@@ -37,12 +37,20 @@ function attachFile( target, file ) {
 function loadVersion( owner, repo, versionPaths, version ) {
 	var versionPath = versionPaths[ version ];
 	if ( !versionPath ) {
-		return when.reject( new Error( util.format(
-			'Version "%s" is not installed for "%s/%s"',
-			version,
-			owner,
-			repo
-		) ) );
+		var error = version ?
+			new Error( util.format(
+				'Version "%s" is not installed for "%s/%s"',
+				version,
+				owner,
+				repo
+			) ) :
+			new Error( util.format(
+				'No installed versions exist for for "%s/%s"',
```

Comment:
There is an extra `for` in the error string and the `version` arg would need to be removed so owner/repo shows up properly.